nvflare/fuel/hci/client/api.py

```diff
@@ -455,6 +456,19 @@ def __init__(
         self.service_finder.start(self._handle_sp_address_change)
         self._start_session_monitor()
 
+    def set_command_timeout(self, timeout: float):
+        if not isinstance(timeout, float):
```

Comment:
note that, this means

set_command_timeout(0) will failed since

```
>>> isinstance(0, float)
False

```

Reply:
maybe we should introduce unset_command_timeout() method.